PYME/Acquire/Hardware/Coherent/OBIS.py

```diff
@@ -138,6 +141,35 @@ def SetPower(self, power):
     def GetPower(self):
         return self.power
 
+    def SetOpMode(self,mode):
```

Comment:
Is this something we might encounter on other lasers? Do we need operating mode as part of the general laser interface? I don't think we need to answer those yet. In the interim, I think leaving this as an OBIS specific command should be fine, but it might be worth signalling that it is OBIS specific in the method name - e.g. 
```suggestion
    def set_obis_operating_mode(self,mode):
```